docs/user/getting-started.md

```diff
@@ -154,42 +68,12 @@ $ kubectl get pods -n flightctl
 
 ### flightctl in ACM
```

Comment:
Now the section that explains how to enable the UI  in ACM feels out of context. 
Users will get to the part about "After deploying the FC ACM UI plugin..." even when they didn't realise that the UI was being installed.

I would make it explicit whether users want to install the UI, and only when they chose to, we include the section about enabling the UI in ACM.

Reply:
Let's keep in mind the "Getting Started" is really about a quick start with maximal likelihood of success for users who just want to kick the tires. Let's therefore focus on a highly opinionated deployment: standalone FC on Kind with built-in auth. We'll have more in-depth docs on other configuration and deployment options.

Therefore, I'd suggest we move the content on UI in ACM etc. to "Installing and Configuring the Flight Control Service and UI".

Reply:
Lets create a followup. I've updated the deployment instructions but I did not change the structure of the docs in this PR.